products/filestore/api.yaml

```diff
@@ -132,6 +138,52 @@ objects:
                 File share capacity in GiB. This must be at least 1024 GiB
                 for the standard tier, or 2560 GiB for the premium tier.
               required: true
+            - !ruby/object:Api::Type::Array
+              name: 'nfsExportOptions'
+              description: |
+                Nfs Export Options. There is a limit of 10 export options per file share.
+              max_size: 10
+              min_version: beta
+              item_type: !ruby/object:Api::Type::NestedObject
+                properties:
+                  - !ruby/object:Api::Type::Array
+                    name: 'ipRanges'
+                    description: |
+                      List of either an IPv4 addresses in the format {octet 1}.{octet 2}.{octet 3}.{octet 4} or CIDR ranges
```

Comment:
This might be copy-pasted but it's a little overly formal to me - I'd say "`xxx.xxx.xxx.xxx` or `xxx.xxx.xxx.xxx/yy`", or "an IPv4 address, or range in CIDR notation".

Reply:
Oh, but feel free to submit without addressing - or after addressing without another review - totally at your discretion.